corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/collections.js

```diff
@@ -25,6 +29,12 @@ hqDefine("cloudcare/js/formplayer/menus/collections", [
         });
     }
 
+    function enableLanguageOption(response, collection) {
+        response.langs = response.locales.slice(1);
```

Comment:
`response.locales` will always be ["default", "whatever other languages are available to the app"]. I'm omitting the first "default" so that the list can be used to populate the language options in the hamburger menu later on